code/modules/transport/pods/warp_travel.dm

```diff
@@ -144,4 +173,182 @@ var/global/list/warp_beacons = list() //wow you should've made one for warp beac
 		playsound(src.loc, "warp", 50, 1, 0.2, 1.2)
 		do_teleport(M, src.target, 1) ///You will appear adjacent to the beacon
 
+/obj/warp_beacon/proc/startpack()
+	src.packable = 0
+	src.icon_state = "beaconpack"
+	SPAWN_DBG(14)
```

Comment:
actionbars or doafter, don't just use a spawn here

Reply:
Resolved with 693bfd0 - also affects the other one